OutSystems/outsystems-datagrid-reactive
# RGRIDT-1063: Adding conditional format to text columns

This PR adds conditional format to text columns

### What was done
* Created ColumnConfigConditionalFormat to be used by columns that don't have editor.
* Added new rules on ConditionalFormat class for text columns

### Test Steps
1. Cells containing 'black' should have background-yellow and their rows should be blue.
2. If you change the Text class input to background-red, the cells should change as well.


### Screenshots
![textcolumn](https://user-images.githubusercontent.com/3113318/143236744-65d902eb-d380-46f1-956d-52748cf1d819.gif)
